### PR TITLE
fix(l10n): fix string extraction with latest jsxgettext

### LINKED ISSUE #4406: String extraction cannot extract "const"
```
Running "jsxextract" task
fxa-content-server.CRITICAL: uncaughtException SyntaxError: Unexpected token (7:6)
  at raise (/Users/vladikoff/mozilla/fxa-local-dev/fxa-content-server/node_modules/jsxgettext-recursive/node_modules/jsxgettext/node_modules/acorn/acorn.js:236:15)
  at unexpected (/Users/vladikoff/mozilla/fxa-local-dev/fxa-content-server/node_modules/jsxgettext-recursive/node_modules/jsxgettext/node_modules/acorn/acorn.js:1094:5)
  at semicolon (/Users/vladikoff/mozilla/fxa-local-dev/fxa-content-server/node_modules/jsxgettext-recursive/node_modules/jsxgettext/node_modules/acorn/acorn.js:1080:47)
  at parseStatement (/Users/vladikoff/mozilla/fxa-local-dev/fxa-content-server/node_modules/jsxgettext-recursive/node_modules/jsxgettext/node_modules/acorn/acorn.js:1350:9)
  at parseTopLevel (/Users/vladikoff/mozilla/fxa-local-dev/fxa-content-server/node_modules/jsxgettext-recursive/node_modules/jsxgettext/node_modules/acorn/acorn.js:1124:18)
  at Object.exports.parse (/Users/vladikoff/mozilla/fxa-local-dev/fxa-
```

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #4406

@shane-tomlinson r?

Meanwhile filed https://github.com/zaach/jsxgettext-recursive/pull/7

### CODE REVIEW

File: grunttasks/l10n-extract.js [hunk 1]
```diff
@@ -22,10 +22,13 @@ module.exports = function (grunt) {
 
     var clientWalker = extract({
       'input-dir': path.join(__dirname, '..', '.es5', 'scripts'),
-      'join-existing': false,
+      joinExisting: true,
       'keyword': ['t', 'unsafeTranslate'],
       'output': 'client.pot',
+      // output-dir is used by jsxgettext-recursive
```
Comment (shane-tomlinson) [inline grunttasks/l10n-extract.js]: Looks like @zaach merged your PR for `outputDir` - https://github.com/zaach/jsxgettext-recursive/pull/7, thanks @zaach!

File: grunttasks/l10n-extract.js [hunk 2]
```diff
@@ -22,10 +22,13 @@ module.exports = function (grunt) {
 
     var clientWalker = extract({
       'input-dir': path.join(__dirname, '..', '.es5', 'scripts'),
```
Comment (shane-tomlinson) [inline grunttasks/l10n-extract.js]: With the newest jsxgettext, do we need to run this on the babelified scripts? Can we skip the babel step?

**Comment by shane-tomlinson** [comment]
Aside from my comments, looks great! No problems that I can see extracting. 👍

File: grunttasks/l10n-extract.js (hunk 2 above)
Comment (vladikoff) [inline grunttasks/l10n-extract.js]: yeap! sgtm!

File: grunttasks/l10n-extract.js (hunk 1 above)
Comment (vladikoff) [inline grunttasks/l10n-extract.js]: 👍

**Pull request merged**